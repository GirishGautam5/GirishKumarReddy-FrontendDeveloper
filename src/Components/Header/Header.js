import React from "react";
import { Link } from "react-router-dom";
import { SiSpacex } from "react-icons/si";
import "./HeaderStyles.css";

export default function Header() {
  return (
    <>
      <header className="header top-0 left-0 p-5 flex items-center justify-between w-full lg:py-0">
        <Link to="/">
          <SiSpacex className="text-8xl text-white" />
        </Link>
        <nav className="navbar">
          <ul>
            <li>
              <Link to="/" className="text-white text-sm lg:text-base">
                Capsules
              </Link>
            </li>
            <li>
              <Link to="/rockets" className="text-white text-sm lg:text-base">
                Rockets
              </Link>
            </li>
          </ul>
        </nav>
      </header>
    </>
  );
}
