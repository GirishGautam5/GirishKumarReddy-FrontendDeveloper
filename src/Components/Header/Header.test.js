import { shallow } from "enzyme";
import Header from "./Header";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("Header Component test suite", () => {
  it("Header Component test case", () => {
    const headerComponent = shallow(<Header />);
    expect(headerComponent).toMatchSnapshot();
  });
});
