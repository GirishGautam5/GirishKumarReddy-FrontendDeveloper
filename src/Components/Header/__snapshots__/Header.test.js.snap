// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component test suite Header Component test case 1`] = `
<Fragment>
  <header
    className="header top-0 left-0 p-5 flex items-center justify-between w-full lg:py-0"
  >
    <Link
      to="/"
    >
      <SiSpacex
        className="text-8xl text-white"
      />
    </Link>
    <nav
      className="navbar"
    >
      <ul>
        <li>
          <Link
            className="text-white text-sm lg:text-base"
            to="/"
          >
            Capsules
          </Link>
        </li>
        <li>
          <Link
            className="text-white text-sm lg:text-base"
            to="/rockets"
          >
            Rockets
          </Link>
        </li>
      </ul>
    </nav>
  </header>
</Fragment>
`;
